iceoryx_hoofs/include/iceoryx_hoofs/posix_wrapper/posix_access_rights.hpp

```diff
@@ -41,21 +41,22 @@ struct PosixRights
 class PosixGroup
 {
   public:
-    using string_t = cxx::string<100>;
+    static constexpr uint64_t MAX_GROUP_NAME_LENGTH = 128;
```

Comment:
Out of curiosity: Why did you choose 128 for the max group name length and 32 for the max user name length?

Reply:
See `man useradd` there it is stated that the max username length is 32.
The maximum length of a group is 128.

Reply:
Got it for the user name but can you provide me a link for the max group name length? Maybe I mix something up here but when I look for instance at `groupadd` documentation, it states that group names can only be up to 16 characters long.